apps/web/src/components/common/sidebar/sections/workspaces/components/create-workspace-modal.tsx

```diff
@@ -38,8 +38,26 @@ export function CreateWorkspaceModal({
     });
   };
 
+  const resetWorkspaceState = async () => {
+    await queryClient.invalidateQueries({ queryKey: ["workspaces"] });
+    setName("");
+  };
+
+  const resetAndCloseModal = () => {
+    resetWorkspaceState();
+    onClose();
+  };
+
+  useEffect(() => {
+    if (open) {
+      requestAnimationFrame(() => {
+        workspaceInputRef.current?.focus();
+      });
+    }
+  }, [open]);
```

Comment:
suggestion: Same here, I believe we can add the `autoFocus` prop to the input and focus it that way.